package.json

```diff
@@ -20,13 +20,13 @@
   "homepage": "https://github.com/matthew-andrews/isomorphic-fetch/issues",
   "dependencies": {
     "node-fetch": "^1.0.1",
-    "whatwg-fetch": ">=0.10.0"
+    "whatwg-fetch": "niftylettuce/fetch"
```

Comment:
because they use `self` instead of `window` and tests don't pass when you
try to simulate browser with jsdom/jsdomify.  though I could be wrong - I
have to rewrite with pure `jsdom` instead of using `jsdomify` per
https://github.com/tmpvar/jsdom/issues/1203#issuecomment-160725194

On Mon, Nov 30, 2015 at 1:58 PM, Matt Andrews <notifications@github.com>
wrote:

> In package.json
> <https://github.com/matthew-andrews/isomorphic-fetch/pull/54#discussion_r46185957>
> :
>
> > @@ -20,13 +20,13 @@
> >    "homepage": "https://github.com/matthew-andrews/isomorphic-fetch/issues",
> >    "dependencies": {
> >      "node-fetch": "^1.0.1",
> > -    "whatwg-fetch": ">=0.10.0"
> > +    "whatwg-fetch": "niftylettuce/fetch"
>
> for various reasons I'd prefer to keep this on the main version — not on
> someone's fork. Why is this needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matthew-andrews/isomorphic-fetch/pull/54/files#r46185957>
> .
>